drivers/i2c/Kconfig.nrfx

```diff
@@ -0,0 +1,115 @@
+# Kconfig - nrfx I2C support
+#
+# Copyright (c) 2017 - 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig I2C_NRFX
+	bool "nRF TWI nrfx drivers"
+	depends on SOC_FAMILY_NRF
+	help
+	  Enable support for nrfx TWI drivers for nRF MCU series.
+	  Peripherals with the same instance ID cannot be used together,
+	  e.g. I2C_0 and SPI_0. You may need to disable SPI_0 or SPI_1.
+
+if I2C_NRFX
+
+config I2C_NRFX_PINS_OVERRIDE_DTS
```

Comment:
Wouldn't `I2C_NRF_OVERRIDE_DTS_PINS` be more adequate (NRF, because this entry, like pins settings, is not nrfx related but generic for nRF chips)?

Reply:
Changed